packages/fe-fpm-writer/test/unit/section.test.ts

```diff
@@ -21,11 +21,11 @@ describe('CustomSection', () => {
             { version: 1.86, expected: join(root, 'section', '1.86') }
         ];
         test.each(testInput)('get root path of template', ({ version, expected }) => {
-            expect(getManifestRoot(root, version)).toEqual(expected);
+            expect(getManifestRoot(version)).toEqual(expected);
         });
         test('invalid version', () => {
             try {
-                getManifestRoot(root, 1.8);
+                getManifestRoot(1.8);
```

Comment:
It is not a problem but why did you change this test?

Reply:
`getManifestRoot`, which gets the manifest template root, no longer requires the templates root path to be passed in as it internally uses the new `getTemplatePath` function.

Reply:
Thanks!